src/tests/rememberMe.test.js

```diff
@@ -0,0 +1,46 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import localStorage from 'localStorage';
+import app from '../app';
+import mockData from './mockData';
+
+chai.use(chaiHttp);
+chai.should();
+
+const rememberMe = () => {
+  describe('Remember user who ckecked to be remembered ', () => {
+    it('it should return 200 when profile information strored in cookies successfull  ', (done) => {
+      chai
+        .request(app)
+        .get('/api/v1/remembered')
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(200);
```

Comment:
You can also test data in cookies to make sure the `req.cookies` is not empty or anything.

Reply:
I have changed form cookies to sessions and I managed to test it,  you can check for now

Reply:
Well done @EugeneMunya